### ISSUE TITLE
problem visualising image for annotation

### ISSUE THREAD

**Issue by kabikaj** [kcl-ddh/digipal#51]
Hi, we've been having problems visualising the images we uploaded for annotation. Most of the times we cannot see the image or we see only part of it. if we zoom out completely we can see the whole image in small size, but the moment we try to zoom in again, the image dissapears. I tried with different browsers (firefox, chrome, opera, windows edge), in windows, linux and android and with low and high resolution images. The problem persisted. In the end, I converted the dimention of one of our images into the one that comes as an example with the archrtype fresh installation, i.e. 1649x2000, and I could see the image fine and zoom in and out without problems. Does this mean that we are force to convert all our images to these dimentions? Cheers! Alicia.

**Comment by pastokes** [comment]
This sounds like a long-standing problem we've had with the IIP image server that we've never managed to resolve despite a lot of effort from us and the IIP folk. When you say it 'disappears', do you mean you see nothing, or that you see pink boxes? In any case there's no fixed image size, but some images particularly smaller ones do have this problem, so perhaps your resizing was an accidental workaround.

**Comment by kabikaj** [comment]
Sometimes I don't see anything and something I see pink boxes. So I guess it is more likely that the problem is gone if the image is large?

**Comment by geoffroy-noel-ddh** [comment]
Yes, I've seen that bug many times before. I believe the combination of tiled Open Layers and the IIP Image server don't work well together with images that have a dimension lower than ~1000 pixel. It is thus recommended to resize a small image before uploading it.
